assets/src/components/Sidebar/CampaignSidebar.js

```diff
@@ -111,6 +111,15 @@ export class CampaignSidebar extends Component {
       ) {
         const theme = await loadTheme(themeName);
         this.setState({ theme: theme });
+        if (themeName) {
+          try {
+            const response = await fetch(`${ themeJsonUrl + themeName.replace('-new', '') }.json`);
+            const theme = await response.json();
+            applyChangesToDom(theme, []);
+          } catch (e) {
+            console.log('Failed loading config for ' + themeName);
```

Comment:
Another approach could be
`console.log('Failed loading config for %s', themeName);`